src/SiSenseWriter.php

```diff
@@ -142,27 +142,44 @@ private function createRelationships(Datamodel $datamodel, Dataset $sourceDatase
 
             $sourceColumn = array_values(
                 array_filter($sourceTable->getColumns(), function (TableColumn $column) use ($relationship) {
-                    return $relationship['column'] === $column->getOid();
+                    return $relationship['column'] === $column->getName();
                 })
             );
 
+            if (count($sourceColumn) === 0) {
+                throw new UserException(sprintf(
+                    'Column "%s" in table "%s" does not exists',
+                    $relationship['column'],
+                    $sourceTable->getName()
+                ));
+            }
+
             $destinationColumn = array_values(
                 array_filter($destinationTable->getColumns(), function (TableColumn $column) use ($relationship) {
-                    return $relationship['target']['column'] === $column->getOid();
+                    return $relationship['target']['column'] === $column->getName();
                 })
             );
+
+            if (count($destinationColumn) === 0) {
+                throw new UserException(sprintf(
+                    'Column "%s" in table "%s" does not exists',
+                    $relationship['target']['column'],
+                    $destinationTable->getName()
+                ));
+            }
+
             $this->logger->info('Creating relationship');
             $this->api->createRelationship(
                 $datamodel->getOid(),
                 [
                     'dataset' => $sourceDataset->getOid(),
                     'table' => $sourceTable->getOid(),
-                    'column' => $sourceColumn[0]['oid'],
+                    'column' => $sourceColumn[0]->getOid(),
                 ],
                 [
                     'dataset' => $destinationDataset->getOid(),
                     'table' => $destinationTable->getOid(),
-                    'column' => $destinationColumn[0]['oid'],
```

Comment:
@ondrajodas je tam nejaky test na tuto cast kodu? IMHO asi nie, ked testy prechadzali a bol tam preklep, neda sa tam nejaky test napisat? (nevyznam sa v tom podrobne, tak vopred prepac ak blba otazka).

Reply:
přemýšlel jsem nad ním jak ho napsat :( ona tam totiž už jedna tabulka musí být založená aby šla vytvořit vazba a nevím úplně jak spustit 2 configy v datadir testech, ale asi to budu muset nějak pořešit, protože máš pravdu

Reply:
@ondrajodas ono by sa to dalo asi nejak takto:
https://github.com/keboola/sisense-writer/pull/8/files

Reply:
@webrouse to já právě Api použít nemůžu protože tam potřebuji třídu Config ale napadlo mě trochu jiné řešení...

Reply:
je to nahrané + jsem přidal ještě test na úspěšně vytvořenou vazbu